### ISSUE TITLE
Communication between host's container and kubernetes stack failed after removing ipsec sidekick containers

### ISSUE THREAD

**Issue by galal-hussein** [rancher/rancher#8364]
**Rancher Versions:**
Server: 1.4.3
kubernetes (if applicable): 1.5.2

**Steps to Reproduce:**
- Start kubernetes on 3 hosts
- On the third host remove the ipsec-router and ipsec-cni-driver containers
- Wait until the containers recreated

**Results:** 
All containers in this host were not able to reach any kubernets container, and kubernetes reported the node as NotReady.

Logs from the ipsec-router container:
```
3/31/2017 10:20:03 PMtime="2017-04-01T05:20:03Z" level=error msg="Failed to add policy: {Dst: 10.42.37.113/32, Src: 10.42.0.0/16, Proto: 0, DstPort: 0, SrcPort: 0, Dir: dir out, Priority: 10000, Index: 0, Mark: <nil>, Tmpls: [{Dst: 104.197.126.97, Src: 10.42.99.8, Proto: esp, Mode: tunnel, Spi: 0x0, Reqid: 0x4d2}]}, file exists"
3/31/2017 10:20:03 PMtime="2017-04-01T05:20:03Z" level=error msg="Failed to add policy: {Dst: 10.42.0.0/16, Src: 10.42.37.113/32, Proto: 0, DstPort: 0, SrcPort: 0, Dir: dir in, Priority: 10000, Index: 0, Mark: <nil>, Tmpls: [{Dst: 10.42.99.8, Src: 104.197.126.97, Proto: esp, Mode: tunnel, Spi: 0x0, Reqid: 0x4d2}]}, file exists"
3/31/2017 10:20:03 PMtime="2017-04-01T05:20:03Z" level=error msg="Failed to add policy: {Dst: 10.42.0.0/16, Src: 10.42.37.113/32, Proto: 0, DstPort: 0, SrcPort: 0, Dir: dir fwd, Priority: 10000, Index: 0, Mark: <nil>, Tmpls: [{Dst: 10.42.99.8, Src: 104.197.126.97, Proto: esp, Mode: tunnel, Spi: 0x0, Reqid: 0x4d2}]}, file exists"
3/31/2017 10:20:03 PMtime="2017-04-01T05:20:03Z" level=error msg="Error reloading backend after receiving the db change: file exists"
3/31/2017 10:20:18 PMtime="2017-04-01T05:20:18Z" level=info msg="Metadata OnChange received, version: 164557-5cc6db6887beb3525be1bd696730805e"
3/31/2017 10:20:18 PMtime="2017-04-01T05:20:18Z" level=info msg=Reconfiguring
```

**Comment by deniseschannon** [comment]
Available in rancher/net:v0.11.01 (ipsec template version `v0.0.5`)  and rancher/network-manager:v0.6.0 (network-services template version `v0.1.2`)

**Comment by galal-hussein** [comment]
Rancher server: v1.5.4-rc1
rancher/net:v0.11.0
rancher/network-manager:v0.6.0 
Kubernetes: v1.5.4-rancher1-1

After deleting the ipsec sidekick containers, the communication broke until the ipsec sidekicks and kubernetes recovered, and then it worked after that successfully.
